CHANGELOG.md

```diff
@@ -0,0 +1,91 @@
+ - Add icons to Windows and MacOS bundles (#498)
+ - Replace OS-specific filesystem functions by C++17 std::filesystem in the CFile and DDIO modules (#383, #443, #475, #484, #486, #499).
+ - Use standard temporary directory for cache files instead of game data directory (#500)
+ - Implement new byteswap functions (#85, #165, #172)
+ - Build the internal level editor using MFC UI framework. The executable is not included in this release because it is still very unstable and not usable, and can only run on Windows (32-bit) (#397, #476)
+ - Render the game to an OpenGL Frame Buffer Object at the game's resolution (#437)
+ - Format code-base using clang-format (#13, #16, #19, #24, #51, #438)
+ - Use upstream libacm 1.3 for ACM audio decoding (#135, #191, #221) 
+ - Implement a new MVE video decoder based on the open source implementation used in D2X (#289). MVE movies for intro and cutscenes can now be played on all platforms. OGV movies shipped with the Linux game cannot be played yet.
+ - Use the "push" model of filling sound buffer for MVE playback (#491)
+ - Write a new implementation of the md5 algorithm used to compute checksums. (#316)
+ - Use VCPKG to build third-party dependencies not included in the code tree, such as SDL2 and zlib (#137, #430, #469, #512).
+ - Set up automated build and testing using Github Actions for Linux (clang/gcc), Windows (MSVC) and MacOS (apple-clang, x64 & arm64) (#14, #28, #29, #45, #50, #60, #123, #133, #139, #155, #166, #177, #182)
+ - Modernize build system: require CMake 3.20, use present-day CMake practices (#20, #63, #77, #86, #99, #104, #127, #131, #144, #220, #225, #227, #235, #256, #318, #340, #400, #407, #422, #460, #463, #482, #504)
+ - Ensure compatibility with popular compilers, convert misc. pieces of code to modern C++ (#17, #20, #23, #40, #54, #58, #70, #108, #304, #328, #344, #358, #388, #405, #416)
+ - Mark relevant variables and functions as static (#185)
+ - Update the module system: isolate code for independant sub-modules in separate directories and CMake targets (#76, #89, #91, #195, #367, #374, #448, #470, #502)
+ - Document build procedure and usage on all platforms (#116, #118, #119, #124, #147, #154, #170, #174, #175, #184, #190, #204, #210, #211, #222, #259, #261, #270, #285, #297, #384, #413, #435)
+ - Unifiy Windows & Unix command line arguments: --foo, -foo and +foo is accepted on any OS (#199)
+ - Refactor CleanupStr in misc/pstring.cpp (#207)
+ - Change non-specific types (e.g. unsigned int) to their stdint specified counterparts (e.g. `uint32_t`) (#268, #464)
+ - Support case-insensitive loading of content files (#294)
+ - Build TCP/IP and PXO multiplayer modules (#296, #302, #420, #424, #432)
+ - Refactor renderer: move non-OpenGL specific code to a base file, abstract triangle rendering code (#315, #323, #336)
+ - Save screenshots in PNG format instead of the legacy TGA format (#325, #348)
+ - Bump max cells value to increate render distance for outdoor terrain (#338)
```

Comment:
Increment?

Reply:
Unless I missed it, this does not mention the PXO restoration, which really ought to be mentioned since https://pxo.nottheeye.com picked up Descent 3 without hesitating. :)